### DISCUSSION TITLE
Equation Result

### DISCUSSION THREAD

**Discussion by cbarcellos-rbi** [Qalculate/qalculate-gtk#482] [General]
I've tried to solve an equation that only can be solved by approximation (e.g. Newton, bisection, etc.). Qalq sends a response that I could not understand, and is not the solution (as far as I understood). I evaluated this in my HP calculator and the value is approximately 2.4818 (which solves the equation)

This is the equation I'm trying to solve:

"x(5985/(sin(x/2))=15700"

The result I'm getting is the following:

x × ((5985 / sin((x / 2) radians)) = 15700) ≈ (x ≈ 12.56637061n + 5.500975342 or x ≈ 12.56637061n + 0.7822099649)x

I'm running version 4.6.1.

Would somebody explain what are these numbers? For instance, what is the `n' after the number?

Thanks in Advance

**Comment by hanna-kn** [comment]
A parenthesis is missing in the expression (before the equals sign) which is therefore interpreted as _x_ multiplied by the equation `(5985/(sin(x/2))=15700` (with the solutions `12.56637061n + 5.500975342` and `12.56637061n + 0.7822099649`, where _n_ represents any integer).

Qalculate is not able to solve `x(5985/(sin(x/2)))=15700` using the ordinary method, but it can be solved using the `newtonsolve` function (e.g. `newtonsolve(x(5985/(sin(x/2)))=15700, 1)`, or `newtonsolve(x(5985/(sin(x/2)))=15700, -1)` for the negative solution).

**Reply by cbarcellos-rbi** [reply]
Thank you for the prompt response!
